test/test-scalastyle-config.xml

```diff
@@ -3,7 +3,7 @@
     <check level="warning" class="org.scalastyle.file.FileTabChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.file.FileLengthChecker" enabled="true">
         <parameters>
-            <parameter name="maxFileLength"><![CDATA[800]]></parameter>
+            <parameter name="maxFileLength"><![CDATA[900]]></parameter>
```

Comment:
CustomsFinancialsApiConnectorSpec became quite large. We might want to consider reducing redundancy between tests. Can be a part of tech debt ticket to address this

Reply:
Yes, that is the plan. Will address this as part of tech debt ticket and revert the maxFileLength to 800.